core/src/main/java/com/linecorp/armeria/server/healthcheck/HealthCheckServiceBuilder.java

```diff
@@ -50,9 +50,19 @@ public final class HealthCheckServiceBuilder implements TransientServiceBuilder
     private AggregatedHttpResponse healthyResponse = AggregatedHttpResponse.of(HttpStatus.OK,
                                                                                MediaType.JSON_UTF_8,
                                                                                "{\"healthy\":true}");
```

Comment:
Would it break backward compatibility if we use `"{\"healthy\":true,\"status\":\"HEALTHY\"}"` for `healthyResponse`?

Reply:
I think it's OK adding a new field to a scheme isn't a breaking change in JSON world.

Reply:
I've added status field 🙇